### DISCUSSION TITLE
<title>https://github.com/Emre033/Discord-Key-Logger</title>

### DISCUSSION THREAD

**Discussion by Xjectro** [community/community#57154] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

He took my codes and transferred them directly to him, I want to complain about it, what should I do?

**Comment by DSDmark** [comment]
Hi @deverays,

Here is offical documentation about reporting, it may help you:- [**GitHub docs**](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-contact-link-abuse-in-the-template-chooser).

And You can report that user.

![image](https://github.com/community/community/assets/85744816/dee2c360-1738-4fc4-8331-b7d70a47285c)

And remove your that `<title>https://github.com/Emre033/Discord-Key-Logger</title>`  title and add that link repo link into your description and write some understandable title like **how to report or take action against user who stole repo code**.

**Reply by Xjectro** [reply]
thx
